### PR TITLE
Run initial functionality on mobile

### PR DESCRIPTION
**Pull request by siepra**

### CODE REVIEW

File: packages/mobile/src/components/Chat/Chat.stories.tsx [hunk 1]
```diff
@@ -15,43 +14,169 @@ storiesOf('Chat', module).add('Default', () => (
...
-      },
-      {
-        id: '4',
-        type: MessageType.Basic,
-        message:
-          'Chupa chups soufflé danish. Cake chocolate brownie cookie bear claw soufflé. Muffin chupa chups sweet caramels jelly beans chocolate bar bonbon liquorice.',
-        createdAt: 0,
-        date: '1:32pm',
-        nickname: 'bartek'
+    messages={{
```
Comment (holmesworcester) [inline packages/mobile/src/components/Chat/Chat.stories.tsx:17]: is there any value in sharing this mock conversation, with all our examples of problematic or edge case messages, between mobile and desktop? it seems good to centralize our trickiest examples

File: packages/mobile/src/components/Chat/Chat.component.tsx [hunk 2]
```diff
@@ -126,3 +132,21 @@ const customInputStyle = StyleSheet.create({
...
   }
 })
+
+export const ChannelMessagesComponent: React.FC<ChannelMessagesComponentProps> = ({ messages }) => {
+  return (
+    <>
+      {Object.keys(messages).map(day => {
+        return (
+          <View key={day}>
+            {/* <MessagesDivider title={day} /> */}
```
Comment (vinkabuki) [inline packages/mobile/src/components/Chat/Chat.component.tsx:142]: Do we need that comment here?

File: packages/mobile/src/components/Chat/Chat.stories.tsx (hunk 1 above)
Comment (siepra) [reply packages/mobile/src/components/Chat/Chat.stories.tsx:17]: Yeah, we could... I'll open separate ticket for that

**Review by vinkabuki — APPROVED** [review]

File: packages/mobile/src/components/Chat/Chat.component.tsx (hunk 2 above)
Comment (siepra) [reply packages/mobile/src/components/Chat/Chat.component.tsx:142]: I think so. Just to remind us that we should place MessagesDivider component (as soon as we add it) in this exact place

Comment (siepra) [reply packages/mobile/src/components/Chat/Chat.component.tsx:142]: I needed to adapt this component so we're able to display messages without having to change the nectars' selectors

**Review by EmiM — APPROVED** [review]

**Pull request merged**